zipkin-collector-service/config/collector-dev.scala

```diff
@@ -13,13 +13,25 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+import com.twitter.app.App
+import com.twitter.util.Future
 import com.twitter.zipkin.anormdb.{SpanStoreBuilder, AggregatesBuilder}
+import com.twitter.zipkin.receiver.kafka.KafkaSpanReceiverFactory
 import com.twitter.zipkin.storage.anormdb.DB
 import com.twitter.zipkin.collector.builder.CollectorServiceBuilder
 import com.twitter.zipkin.storage.Store
+import com.twitter.zipkin.thriftscala
 
 val db = DB()
 
 val storeBuilder = Store.Builder(SpanStoreBuilder(db, true), AggregatesBuilder(db))
 
-CollectorServiceBuilder(storeBuilder)
+// experimental support for kafka. copy-pasted temporarily into collector-*
+val rcv = sys.env.get("KAFKA_ZOOKEEPER_CONNECT").map(zookeeperConnect => {
```

Comment:
could someone have `KAFKA_ZOOKEEPER_CONNECT` set (for something else) and activate this by mistake?